### PR TITLE
create rest API for rangeallocations.security.openshift.io

### PR DESCRIPTION
**Pull request by deads2k**
Need this API to support removing direct etcd access from controllers: https://github.com/openshift/origin/pull/19203 .

@liggitt you approved the API already.  This just plumbs it through.  Notice that I only do objectmeta validation on these types.  The rangeallocation is intentionally opaque and based on a meta API.

### CODE REVIEW

File: pkg/cmd/server/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -189,6 +189,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule(read...).Groups(networkGroup, legacyNetworkGroup).Resources("clusternetworks", "egressnetworkpolicies", "hostsubnets", "netnamespaces").RuleOrDie(),
 
 				rbac.NewRule(read...).Groups(securityGroup, legacySecurityGroup).Resources("securitycontextconstraints").RuleOrDie(),
+				rbac.NewRule(read...).Groups(securityGroup).Resources("rangeallocations").RuleOrDie(),
```
Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/policy.go]: will regenerate policy yaml

Comment (deads2k) [reply pkg/cmd/server/bootstrappolicy/policy.go]: > liggitt 2 minutes ago  Member
> will regenerate policy yaml

done

**Pull request merged**